pkg/config/validation/validation.go

```diff
@@ -1390,10 +1390,6 @@ func ValidateDrainDuration(drainTime *durationpb.Duration) (errs error) {
 			errors.New("drain time only supports durations to seconds precision"))
 	}
 
-	if drainDuration > drainTimeMax {
```

Comment:
Why it was there?

If  remove, drainTimeMax should be removed too

Reply:
Do not the know the history. But seems not needed. Removed the var also